README.md

```diff
@@ -1,46 +1,148 @@
+       1. do they fit your skills/ goals
+       1. be prepared to talk about them/ discuss complexity/ estimates
+ 1. [PO]
+    1. [Stakeholder Backlog] see above: check priority and well defined; assign or label or/and move to [Product Backlog]
+    1. [Done] see above: inform `Stakeholder` about the solved issues and request 
+
+### Sprint
+The [Sprint] is the time period between Sprint Planning and Sprint Review.
+We will use a 2 weeks period.
+We combine the Sprint Planning and the Sprint Review into one meeting at the beginning of every other week.
+During the [Sprint] the assignees work on their issues that are in the [Sprint Backlog].
+As a rule of thumb, an issue needs to have 13 or less Story Points (see [Glossary] & [Story Points Estimates]) to be solvable in one [Sprint] (two weeks).
+Therefore an associate can have only 13 or less Story Points alltogether in one [Sprint].
+These are just general guidelines. Over time each team can find its own rules and time estimates ([Velocity])
+
+
+### Approved by Stakeholder
+tbd
+
+#### During the [Sprint]
+tbd
+
+### Redefine Issues & Epics
+Goal of [Redefine Issues & Epics] is to understand issues better if needed and split issues bigger than 13 Story Points (= Epics) into smaller issues (<= 13 Story Points).
+This will lead to issues that can be direct worked on (allow them into the [Product Backlog])
+
+tbd: when, who and how - hands on
+
+---
+
+## Synchronous work
+
+### Sprint Planning / Review meeting
+Goal of the Sprint
+
+#### Review 
+#### Sprint Planning
+
+#### During the meeting
 
 1. issues in the `Done` column with a -1 are discussed and all other issues are closed out
 2. issues in the `Product Backlog` are assigned and moved to the `Sprint Backlog`
 
+#### Sprint Backlog
+
 ## Product Owner
+The [Product Owner] is repsonsible for:
+ - Priorizing issues: Move it from [Stakeholder Backlog] to [Product Backlog]
+ - Well defined: Approve an issue or Request a better defintion --> set `triage/accepted` label
+ - Check or delegate the [Approval by Stakeholder]
 
 @goern and @durandom are product owners
 
-# Further reading
-
-The Thoth team practiced SCRUM and maintains their [Terms and Conditions](https://github.com/thoth-station/core/blob/master/docs/TermsAndConditionsForTheScrum.md), which we can build upon.
\ No newline at end of file
+## Glossary
+PO - Product Owner
+well defined issue - an issue that has enough details, understanding of the problem, that it can be used in the [Product Backlog](#product-backlog)
+Story Points - Effort estimation for an issue.  [Glossary] 
+
+## The Scrum Board
+[SCRUM BOARD]
+| Stakeholder Backlog | Product Backlog | Sprint Backlog | In Progress | Done | Approved by Stakeholder | Closed |
+|---------------------|-----------------|----------------|-------------|------|-------------------------|--------|
+|                     |                 |                |             |      |                         |        |
```

Comment:
not sure I see the point of this markdown table  :thinking:

Reply:
1. to see the board columns without opening the board (I thought it might be easier seeing it on the same page)
2. maybe for adding basic rules in the colums data area, like "must be `triage accepted` or must be assigned ...